pkg/interface/src/views/apps/profile/components/EditProfile.tsx

```diff
@@ -103,28 +144,73 @@ export function EditProfile(props: any): ReactElement {
         initialValues={contact || emptyContact}
         onSubmit={onSubmit}
       >
-      <Form width="100%" height="100%" p={2}>
-        <Input id="nickname" label="Name" mb={3} />
-        <Col width="100%">
-          <Text mb={2}>Description</Text>
-          <MarkdownField id="bio" mb={3} s3={props.s3} />
-        </Col>
-        <ColorInput id="color" label="Sigil Color" mb={3} />
-        <Row mb={3} width="100%">
-          <Col pr={2} width="50%">
-            <ImageInput id="cover" label="Cover Image" s3={props.s3} />
-          </Col>
-          <Col pl={2} width="50%">
-            <ImageInput id="avatar" label="Profile Image" s3={props.s3} />
-          </Col>
-        </Row>
-        <Checkbox mb={3} id="isPublic" label="Public Profile" />
-        <GroupSearch label="Pinned Groups" id="groups" groups={props.groups} associations={props.associations} publicOnly />
-        <AsyncButton primary loadingText="Updating..." border mt={3}>
-          Submit
-        </AsyncButton>
-      </Form>
-    </Formik>
-  </>
+        {({ setFieldValue }) => (
+          <Form width='100%' height='100%'>
+            <ProfileHeader>
+              <ProfileControls>
+                <Row>
+                  <button
+                    type='submit'
+                    style={{ appearance: 'none', background: 'transparent' }}
+                  >
+                    <Text py='2' cursor='pointer' fontWeight='500' color='blue'>
+                      Save Edits
+                    </Text>
+                  </button>
+                  <Text
+                    py='2'
+                    ml='3'
+                    fontWeight='500'
+                    cursor='pointer'
+                    onClick={() => {
+                      history.push(`/~profile/${ship}`);
+                    }}
+                  >
+                    Cancel
+                  </Text>
+                </Row>
+                <ProfileStatus contact={contact} />
+              </ProfileControls>
+              <ProfileImages hideCover={hideCover} contact={contact}>
+                <ProfileHeaderImageEdit
+                  contact={contact}
+                  s3={props.s3}
+                  setFieldValue={setFieldValue}
+                  handleHideCover={handleHideCover}
+                />
+              </ProfileImages>
+            </ProfileHeader>
+            <Row mb={3} pt={5} width='100%'>
+              <Col pr={2} width='25%'>
+                <ColorInput id='color' label='Sigil Color' />
```

Comment:
This throws an error in the console, complaining about a `value` prop being set without an `onChange` handler, but both are present in the `ColorInput` component itself. Has anyone else fought this fight? It doesn't block interaction and doesn't introduce a regression, but adds some unwanted noise in the console.

Reply:
Thanks to @liam-fitzgerald